### PR TITLE
fix(clerk-js): UI fixes to match designs for <SignIn/> & <SignUp/>

### PR DESCRIPTION
**Pull request by octoper**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by octoper** [comment]
!preview

**Comment by octoper** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @octoper, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-r8sz6cfy5.clerkstage.dev) | Jan 16, 2024 12:15 PM |

File: packages/clerk-js/src/ui/elements/ApplicationLogo.tsx [hunk 1]
```diff
@@ -66,7 +66,18 @@ export const ApplicationLogo = (props: ApplicationLogoProps) => {
...
+          sx={{
+            justifyContent: 'center',
+          }}
+          to={logoUrl}
+        >
+          {image}
+        </RouterLink>
+      ) : (
+        image
+      )}
```
Comment (anagstef) [inline packages/clerk-js/src/ui/elements/ApplicationLogo.tsx:80]: @octoper is this centering only if there is a logoUrl? What happens when we only have an image?

**Review by anagstef — APPROVED** [review]
💯 Just remember to add the two localization changes in the clerk-docs PR: https://github.com/clerk/clerk-docs/pull/632

File: packages/clerk-js/src/ui/elements/ApplicationLogo.tsx (hunk 1 above)
Comment (octoper) [reply packages/clerk-js/src/ui/elements/ApplicationLogo.tsx:80]: This already works as we had `justifyContent: 'center'` in the outer element https://github.com/clerk/javascript/pull/2591/files#diff-4a9926d4df94a1d58e1ff0f95a12fa0399f27a30212873d2bc3d96a8842abb0eL57

**Pull request merged**